### PR TITLE
Adds support for collecting libs

### LINKED ISSUE #104: Get an output of a library
I need to get the output of just the linked libraries to use in ffi code. It builds correctly and if I do -K I can see the libraries are created with:
```
  5 myPackages = naersk.buildPackage {                                                
  6     root = ../.;                                                                     
  7     copyBins = true;                                                                 
  8     copyTarget = false;                                                              
  9   };
```

### PR DESCRIPTION
**Pull request by angerman**
Using rust as library provider requires us to collect the
libraries (shared and static) to link against.

Ideally we'd also copy headers, but I didn't find those yet.

Fixes #104

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,6 +62,8 @@ it is converted to an attribute set equivalent to `{ root = theArg; }`.
 | `targets` | The targets to build if the `Cargo.toml` is a virtual manifest. |
 | `copyBins` | When true, the resulting binaries are copied to `$out/bin`. <br/> Note: this relies on cargo's `--message-format` argument, set in the default `cargoBuildOptions`. Default: `true` |
 | `copyBinsFilter` | A [`jq`](https://stedolan.github.io/jq) filter for selecting which build artifacts to release. This is run on cargo's [`--message-format`](https://doc.rust-lang.org/cargo/reference/external-tools.html#json-messages) JSON output. <br/> The value is written to the `cargo_bins_jq_filter` variable. Default: `''select(.reason == "compiler-artifact" and .executable != null and .profile.test == false)''` |
+| `copyLibs` | When true, the resulting binaries are copied to `$out/bin`. <br/> Note: this relies on cargo's `--message-format` argument, set in the default `cargoBuildOptions`. Default: `true` |
```
Comment (disassembler) [inline README.md:65]: $out/lib

Comment (angerman) [reply README.md]: Thanks. Fixed.

**Comment by cleverca22** [comment]
`lndir` is currently fetched from the target when cross-compiling, https://github.com/angerman/naersk/pull/1 fixes it and will add the fix to this PR if merged

**Comment by disassembler** [comment]
@nmattia can we get this merged? Do you have any concerns with it? We're starting to use this in production.

**Pull request merged**